pkg/apis/networking/validation/validation.go

```diff
@@ -213,12 +210,7 @@ func validateIngress(ingress *networking.Ingress, opts IngressValidationOptions,
 func ValidateIngressCreate(ingress *networking.Ingress, requestGV schema.GroupVersion) field.ErrorList {
 	allErrs := field.ErrorList{}
 	var opts IngressValidationOptions
-	opts = IngressValidationOptions{
-		// TODO(robscott): Remove regex validation for 1.19.
-		requireRegexPath: true,
-		// TODO(cmluciano): Allow resource backend for 1.19.
-		allowResourceBackend: false,
-	}
+	opts = IngressValidationOptions{}
```

Comment:
Why leave these options at all? Just in case we make any post-v1 additions to Ingress? Could we choose to add them back at that point instead?

Reply:
mostly to limit the scope of changes and conflicts between this and other ingress PRs... once all the work is done, we can clean up signatures if we end up not needing options